tasks/prime.py

```diff
@@ -119,6 +120,25 @@ def create_mto_shipment(self):
         else:
             logger.info(f"ℹ️ MTOShipment {json_body['id']} created!")
 
+    @tag("paymentRequest", "createUpload")
+    @task
+    def create_upload(self):
+        payment_request_id = "a2c34dba-015f-4f96-a38b-0c0b9272e208"
+        upload_file = {"file": open(TEST_PDF, "rb")}
+
+        resp = self.client.post(
+            prime_path(f"/payment-requests/{payment_request_id}/uploads"), files=upload_file, **self.user.cert_kwargs
```

Comment:
Locust uses the path to group requests, so we need to add a `name` attribute so that the stats for this endpoint will all be grouped together regardless of which ID is passed in. Ex:

```python
resp = self.client.post(
    prime_path(f"/payment-requests/{payment_request_id}/uploads"),
    files=upload_file, 
    name=prime_path("/payment-requests/:paymentRequestID/uploads"),  # matches the swagger path def
    **self.user.cert_kwargs
)
```

This is impossible to notice when testing with a static ID, haha.